packages/quantic/force-app/main/default/lwc/quanticRecommendationList/templates/grid.html

```diff
@@ -0,0 +1,28 @@
+<template>
+  <template lwc:if={recommendations.length}>
+    <div class="slds-text-heading_medium slds-var-p-bottom_x-small">
+      <c-quantic-heading
+        data-cy="recommendations__label"
+        level={headingLevel}
+        label={label}
+      ></c-quantic-heading>
+    </div>
+    <div class="slds-grid recommendations-list__container">
+      <template for:each={recommendations} for:item="recommendation">
+        <div
+          key={recommendation.uniqueId}
+          data-cy="recommendations__item"
+          class="recommendation-item__container slds-var-p-vertical_x-small slds-var-p-right_medium"
+        >
```

Comment:
I didn't want to use the gutters in order to always have full control of the width of each element and avoid adding more calculation  that includes the gutters to determine how many item to display per row.

Reply:
Also the paddings and margins would help support the carousel layout.
because in the carousel it's a bit special, we don't have a regular space between each recommendation:
Example:
`-` represents a space
`()` represents the limits of a carousel page
```
(rec1-rec2-rec3)(rec4-rec5-rec6)
```

Notice that between `rec1` and `rec2` we should have a space, however between `rec3` and `rec4`  we shouldn't.

That's what I mean by saying that we will not always have a regular space between the recommendations, that's what making the padding/margin the best solution to handle this space.

Reply:
Gotcha, okay